diva-irma.js

```diff
@@ -16,6 +16,7 @@ let divaState;
 * @private
 */
 
+const base64 = require('base-64');
```

Comment:
Why a separate package? NodeJS has native support for decoding base64 strings:

```
> console.log(new Buffer("Hello World").toString('base64'));
SGVsbG8gV29ybGQ=
> console.log(new Buffer("SGVsbG8gV29ybGQ=", 'base64').toString('ascii'))
Hello World
```

Reply:
Fixed